topics/single-cell/tutorials/scrna-ncbi-anndata/tutorial.md

```diff
@@ -0,0 +1,521 @@
+---
+layout: tutorial_hands_on
+
+title: Converting NCBI Data to the AnnData Format
+subtopic: datamanipulation
+priority: 3
+zenodo_link: 'https://zenodo.org/record/10101768'
+
+questions:
+- How do I understand NCBI data?
+- How can I convert raw gene data to the AnnData format?
+- How do I manually and automatically add metadata to my AnnData object?
+objectives:
+- Analyse gene data from NCBI.
+- Convert raw gene expression data into the AnnData format.
+- Add metadata to the AnnData object to prepare it for analysis.
+time_estimation: 1H
+key_points:
+- Single cell data often requires additional processing and conversion before it is able to be analysed.
+- Metadata and annotations can be added in both manually and automatically with various tools and resources.
+
+tags:
+- data management
+- data import
+
+contributions:
+  authorship:
+    - hexhowells
+  editing:
+    - nomadscientist
+
+follow_up_training:
+  -
+    type: "internal"
+    topic_name: single-cell
+
+---
+
+The goal of this tutorial is to take raw NCBI data from some published research, convert the raw data into the AnnData format then add metadata to the object so that it can be used for further processing / analysis. Here we will look at the steps to obtain, understand, and manipulate the data in order for it to be properly processed.
+
+# Obtaining the Data
+
+The first step is the obtain the data. For this tutorial, we will use data from the paper {% cite Song2022 %}. The data for this research is stored in the Gene Expression Omnibus (GEO) which is a public repository storing public genomics data.
+
+{% include _includes/cyoa-choices.html option1="Manual" option2="Zenodo" default="Zenodo"
+       text="If you have experience finding and downloading data from GEO then you can use the Zenodo link to load the data directly into Galaxy, if you don't have experience with GEO then you can manually download, prepare, and load the data into Galaxy." %}
+
+
+<div class="Manual" markdown="1">
+> <hands-on-title>Download and extract the data from GEO</hands-on-title>
+>
+> 1. Using a web browser navigate to the GEO repository for the paper
+>     ```
+>     https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE176031
+>     ```
+> 2. Copy the ```(http)``` link located in the supplemental materials section of the page
```

Comment:
```suggestion
> 2. Copy the ```(https)``` link located in the supplemental materials section of the page
```

just nitpicking

Reply:
I've put ```(http)``` as it is displayed that way on the GEO page.